test/fuzz/README.md

```diff
@@ -0,0 +1,78 @@
+# Fuzzing
+
+ Fuzzing or fuzz testing is an automated software testing technique that
+ involves providing invalid, unexpected, or random data as inputs to a
+ computer program. A fuzzing target is defined for running the tests.
+ These targets are defined in the test/fuzz directory and run using
+ the instructions given below:
+
+- Install the package from <https://github.com/mdempsky/go114-fuzz-build>
+  using go get command. This is the link for the fuzzing tool that uses
+  libfuzzer for its implementation purposes. I set it up and made a sample
+  function to run the fuzzing tests. In these steps following are the important steps
```

Comment:
Would it be possible to add a make target for this? With that we would not need so much documentation for how to run it and it would be far easier to use it. Maybe `make fuzz`?

Reply:
I've added a make fuzz to run the target now. so this documentation is updated now.  For running multiple tests in parallel, I am also now looking [here](https://github.com/golang/go/issues/46312#issuecomment-1153345129). In this link a shell script is written to run all the fuzzing targets inside a file one by one which can be a further improvement to the make fuzz command. I'll now try to add this as see if it works.